sdk/js/src/service/devices/index.js

```diff
@@ -185,17 +184,44 @@ class Devices {
     return Marshaler.payloadListResponse('end_devices', response)
   }
 
-  async getById(applicationId, deviceId, selector = [['ids']], { ignoreNotFound = false } = {}) {
-    const response = await this._getDevice(
+  /**
+   * Gets the `deviceId` end device under the `applicationId` application.
+   * This method will assemble the end device from all available stack
+   * components (i.e. NS, AS, IS, JS) based on the provided `selector`
+   * and the end device existence in the respective components.
+   * Note, this method throws an error if the requested end device does not
+   * exist in the IS.
+   * @param {*} applicationId - Application ID
```

Comment:
```suggestion
   *
   * @param {*} applicationId - Application ID
```
Saves me a merge after #2356